charlib/config/syntax.py

```
@@ -476,6 +476,13 @@ class ConfigFile:
                             'keyword is set to ``True``'
             ), default='debug'
         ) : str,
+        Optional(
+            Literal(
+                'dry_run',
+                description='If true, CharLib will perform all steps except for running SPICE ' \
+                            ' simulations. Equivalent to the ``--no-sim`` command line option.'
```

Ruff (0.15.21)

[warning] Implicitly concatenated string literals over multiple lines

(ISC002)